src/FTPCrawler.py

```diff
@@ -102,5 +107,7 @@ def scan(self, directory=None):
                             print directory
                             print item_name
                             #raise( the_e )
-                        wx.CallAfter(self.main_app.process_new_result, result_dict=video_info)
+
+                        wx.PostEvent( self.main_app, FTPMonitorEvents.ResultFoundEvent( video_info ) )

```

Comment:
I ended up creating an event for this because I was getting assert errors the other way.  However, that may have been due to something else...I can change it back to CallAfter if you want